ethers-signers/src/aws/mod.rs

```diff
@@ -50,6 +53,7 @@ pub struct AwsSigner {
     key_id: String,
     pubkey: VerifyingKey,
     address: Address,
+    timeout: Option<Duration>,
```

Comment:
do we want to default to a timeout in the constructor so this is always defined?

Reply:
I don't have a strong preference, I implemented it this way mostly to easily switch back to the old behavior if we ever decide it to be desirable. Unless you feel strongly I'll probably keep as-is just in the interest of time